plugins/bulk-import-backend/src/service/router.test.ts

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  */
```

Comment:
You can refer to the OCM router tests [here](https://github.com/janus-idp/backstage-plugins/pull/2104/files#diff-4327a4f681596c8e0c0869c6cea617aa42ce2226867653ed0190ee0093051254). Going forward, we should avoid manually mocking any services.

Reply:
Done by using `mockServices` from `@backstage/backend-test-utils` as much as possible. See [`04c062a` (#2232)](https://github.com/janus-idp/backstage-plugins/pull/2232/commits/04c062acfbdeece8e8cd92cce7a2e78bfaf7c588). Thanks.